src/Ast/PhpVersion.php

```diff
@@ -0,0 +1,65 @@
+<?php declare(strict_types=1);
+
+namespace DoclerLabs\ApiClientGenerator\Ast;
+
+use UnexpectedValueException;
+
+class PhpVersion
+{
+    public const VERSION_PHP70      = '7.0';
+    public const VERSION_PHP71      = '7.1';
+    public const VERSION_PHP72      = '7.2';
```

Comment:
I would highlight in the changelog that php 7.2 is supported now.

Reply:
done